recipes/cpython/all/conanfile.py

```diff
@@ -102,6 +102,10 @@ def configure(self):
     def layout(self):
         basic_layout(self, src_folder="src")
 
+    def build_requirements(self):
+        if Version(self.version) >= "3.11" and not is_msvc(self) and not self.conf.get("tools.gnu:pkg_config", check_type=str):
```

Comment:
Why is it needed now?

Reply:
CPython started moving some of its dependency management to `pkg-config` as of 3.11, and (more or less) fully in 3.12.